SpywareFilter/sections/specific.txt

```diff
@@ -144,8 +144,11 @@ fategrandorder.info##.sidebar_acr.side_inline
 ||dojq4kt8ws9iq.cloudfront.net/js/analytics.js
 ||components2.gazzettaobjects.it/rcs_tealium/
 ||liquipedia.net/stats/
-||static.olx.com.br/olx/js/lurker.min.js
+||lurker.olx.com.br^
+||geoip.olx.com.br^
+||pub.olx.com.br/listing.js
```

Comment:
What the purpose?

Reply:
This is script carries advertisements and is also what initiates contact with lurker.olx.com.br
https://bin.disroot.org/?72fd632e1069e9ae#2KBqqNrT4V8eVbQbFc4uahW86eNkkF9HudT2qTid8Ty7